### PR TITLE
[snapcast] Adding new chart

### PR DESCRIPTION
**Pull request by lanquarden**
Signed-off-by: lanquarden <lanquarden@gmail.com>

**Description of the change**

Adding snapcast (server) helm chart

**Benefits**

Easily run a snapcast server in your cluster

**Possible drawbacks**

It's not an official docker container from the project, I've picked one that is following the release cadence and versions the containers.

**Applicable issues**

- fixes #

**Additional information**

This is just a bare bones version, manually installed with helm from the repo and it's running. As I learn how to successfully deploy this together with a mopidy chart I'll try to find a way to poor that back into the documentation/values.yaml

My first attempt at creating a helm chart, the common chart and the wealth of examples make this a breeze!

**Checklist** 
- [X] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Chart `artifacthub.io/changes` changelog annotation has been updated in `Chart.yaml`. See [Artifact Hub documentation](https://artifacthub.io/docs/topics/annotations/helm/#supported-annotations) for more info.
- [X] Variables have been documented in the `values.yaml` file.

### CODE REVIEW

**Comment by Truxnell** [comment]
Thanks @lanquarden 

Looks like you just have a little linting to do.
`Error: [colons] too many spaces after colon`

**Comment by lanquarden** [comment]
Linting issue should be fixed now

File: charts/incubator/snapcast/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+appVersion: v0.26.0
+description: snapcast helm package
+name: snapcast
+version: 0.0.1
+kubeVersion: ">=1.16.0-0"
+keywords:
+- snapcast
+- snapserver
```
Comment (Truxnell) [inline charts/incubator/snapcast/Chart.yaml]:
```suggestion
  - snapcast
  - snapserver
```

File: charts/incubator/snapcast/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+home: https://github.com/k8s-at-home/charts/tree/master/charts/incubator/snapcast
+icon: https://github.com/badaix/snapcast/blob/master/doc/web_hi_res_512.png
+sources:
+- https://github.com/badaix/snapcast
```
Comment (Truxnell) [inline charts/incubator/snapcast/Chart.yaml]:
```suggestion
  - https://github.com/badaix/snapcast
```

File: charts/incubator/snapcast/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+maintainers:
+- name: lanquarden
+  email: lanquarden@gmail.com
```
Comment (Truxnell) [inline charts/incubator/snapcast/Chart.yaml]:
```suggestion
  - name: lanquarden
    email: lanquarden@gmail.com
```

File: charts/incubator/snapcast/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+dependencies:
+- name: common
+  repository: https://library-charts.k8s-at-home.com
+  version: 4.4.2
```
Comment (Truxnell) [inline charts/incubator/snapcast/Chart.yaml]:
```suggestion
  - name: common
    repository: https://library-charts.k8s-at-home.com
    version: 4.4.2
```

File: charts/incubator/snapcast/values.yaml [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: saiyato/snapserver
+  # -- image tag
+  # @default -- chart.appVersion
+  tag: v0.26.0
```
Comment (Truxnell) [inline charts/incubator/snapcast/values.yaml:13]:
```suggestion
  tag:
```
This inherits from the `Chart.yaml` so we prefer to leave this blank -  then you only have to maintain version once not twice.

**Review by Truxnell — CHANGES_REQUESTED** [review]
Few minor changes/linting, which i think you may have already done while i was reviewing!

**Pull request merged**